src/base/LockManagerBase.sol

```diff
@@ -70,6 +77,21 @@ abstract contract LockManagerBase is ILockManager {
         return knownProposalIds.length();
     }
 
+    /// @notice Returns how many of the known proposalID's were created by the given address
+    /// @param _creator The address to use for filtering
+    function activeProposalsCreatedBy(address _creator) public view virtual returns (uint256 _result) {
+        uint256 _proposalCount = knownProposalIds.length();
+        for (uint256 _i; _i < _proposalCount; _i++) {
+            uint256 _proposalId = knownProposalIds.at(_i);
+            if (knownProposalIdCreators[_proposalId] != _creator) {
+                continue;
+            } else if (!plugin.isProposalOpen(_proposalId)) {
```

Comment:
Multiplier can be bypassed for proposals that start in the future and aren't open yet.

Reply:
This is taken care of in the follow-up PR, which I will rebase to appear before this one